### ISSUE TITLE
Keyboard navigation for recent files

### ISSUE THREAD

**Issue by pgrit** [pgrit/beampdf#4]
The recent file picker should be keyboard navigable.
Focus should start on the most recent file, and be indicated by a box around it.
Arrow keys and/or tab should be usable to move this selection around.
Enter and/or space should then open the selected file.
